src/Select.js

```diff
@@ -27,10 +34,10 @@ class Select extends Component {
 	 * @protected
 	 */
 	findItemIndex_(element) {
-		var items = this.element.querySelectorAll('li');
-		for (var i = 0; i < items.length; i++) {
-			if (items.item(i) === element) {
-				return i;
+		const items = this.element.querySelectorAll('li');
+		for (const [index, item] of items.entries()) {
+			if (item === element) {
```

Comment:
@brunobasto. How can I test the `else` branch if this is an internal method and it is only fired if a select item exists? I can't find a case that the if never match the expression.